### PR TITLE
Midterm status

### PR DESCRIPTION
**Pull request by dustxd**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (yehee) [inline .vscode/settings.json]: Should we include `.vscode` to `.gitignore` file?

Comment (dustxd) [reply .vscode/settings.json]: I think it's okay to leave it there...We do want the eslint settings to be in the repo, don't we? Or it's some other settings that you're worried about?

Comment (yehee) [reply .vscode/settings.json]: I just don't want to override the setting at some point. Otherwise, it's fine to keep it.

Comment (dustxd) [reply .vscode/settings.json]: Ah I see :) Not sure if Martin wants to make anymore changes in this, so I'm just leaving it there for now :p

File: client/src/newStore.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { applyMiddleware, compose as reduxCompose, createStore } from 'redux';
```
Comment (dustxd) [inline client/src/newStore.js]: @martinjoha Btw I don't see this file being used anywhere, can I just remove it?

Comment (martinjoha) [reply client/src/newStore.js]: its fine to delete it

File: client/src/components/dashboard/OverviewPage.jsx [hunk 3]
```diff
@@ -38,4 +54,8 @@ const mapStateToProps = (state) => ({
   hasTriedLogin: user.hasTriedLogin(state),
 });
 
-export default withStyles(styles)(connect(mapStateToProps)(OverviewPage));
+const mapDispatchToProps = (dispatch) => ({
```
Comment (dustxd) [inline client/src/components/dashboard/OverviewPage.jsx:57]: @martinjoha We don't need to use this, just do the following whenever needed
```const { actions } = this.props;```
```actions.sendEmail(subject, body);```

**Comment by dustxd** [comment]
#39 Merged the change in project folder structure (with Express installation) into master

Separated `midterm-status` branch into `remote-database` and `send-email` branches
#40 `remote-database` branch -- set up remote database and added API endpoints

Closing this PR because this branch was overloaded

**Pull request closed without merging**